src/js/game/LevelMVC/LevelPlane.js

```diff
@@ -311,9 +344,41 @@ module.exports = class LevelPlane extends Array {
       posToRefresh.push(this.redstoneListON[i]);
     }
 
+    // Once we're done updating redstoneWire states, check to see if doors should open/close.
+    // Do things a bit different
+    if (OnDoor) {
+      this.findDoorToAnimate(this.coordinatesToIndex(DoorPosition));
+    } else {
+      this.findDoorToAnimate(-1);
+    }
     return posToRefresh;
   }
 
+  positionEquivalence(lhs, rhs) {
+    if (lhs[0] === rhs[0]) {
+      if (lhs[1] === rhs[1]) {
+        return true;
+      }
+    }
+    return false;
+  }
```

Comment:
is this used anywhere?

Reply:
Good catch! It was, but I refactored the door logic with the the pressure-plate-right-next-to-door scenario, and this was no longer needed 👍